dspace-api/src/main/java/org/dspace/authenticate/IPMatcher.java

```diff
@@ -279,7 +279,11 @@ public boolean match(String ipIn) throws IPMatcherException
             }
             catch (UnknownHostException e)
             {
-                throw new IPMatcherException("Malformed IPv6 address ",e);
+                throw new IPMatcherException("Malformed IPv6 address: " + ipIn,e);
+            }
+            catch (IllegalArgumentException ia)
```

Comment:
IllegalArgumentException  does not seem to be generated from the code within the try block.  Should the call to ip4ToIp6() be included in the try block?